app/models/user.py

```diff
@@ -0,0 +1,24 @@
+from app import db
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+
+class User(db.Model):
+    id = db.Column(db.BigInteger, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(250), nullable=False)
+    email = db.Column(db.String(100), index=True, unique=True, nullable=False)
+    password = db.Column(db.String(250), nullable=False)
+    grade = db.Column(db.String(250), nullable=False)
```

Comment:
does a teach also have grade?
Why is this not nullable?

Reply:
done